Source/SelfService/Web/layout/layoutWithSidebar.tsx

```diff
@@ -101,28 +62,81 @@ export const LayoutWithSidebar: React.FunctionComponent<Props> = (props) => {
 
 export const getDefaultMenuWithItems = (
     history: History<LocationState>,
-    items: any[]
+    topItems: any[],
+    bottomItems: any[]
 ): React.ReactNode => {
     return (
         <>
-            <ul>
-                {items.map((link) => {
+            <List>
+                {topItems.map((link) => {
+                    return (
+                        <ListItemButton
+                            disableGutters
+                            key={link.name}
+                            selected={window.location.href.includes(link.href)}
```

Comment:
This is such a nice, sweet way to resolve the selected state! :)
There are some potential pitfalls if multiple url's have the same paths, which would lead to multiple items being selected, but I don't see this happening any time soon. Let's use this and improve 👍

Reply:
yayyy